examples/sklearn/iris-classifier/trainer.py

```diff
@@ -0,0 +1,24 @@
+# Train the model
+
+from sklearn.datasets import load_iris
+from sklearn.model_selection import train_test_split
+from sklearn.linear_model import LogisticRegression
+
+iris = load_iris()
+data, labels = iris.data, iris.target
+training_data, test_data, training_labels, test_labels = train_test_split(data, labels)
+
+model = LogisticRegression(solver="lbfgs", multi_class="multinomial")
+model.fit(training_data, training_labels)
+accuracy = model.score(test_data, test_labels)
+print("accuracy: {:.2f}".format(accuracy))
+
+
+# Upload the model
+
+import boto3
```

Comment:
Typically this would be at the top, but I can see why it's down here. I still might lean toward it being at the top